spec/helpstation/processors/by_key_fetcher_spec.rb

```diff
@@ -22,7 +22,7 @@
       expect(subject).to be_a(Substation::Response::Success)
       expect(subject.output).to eq(
         my_input: input[:my_input],
-        my_output: output
+        my_output: 'test and {:mock_service=>:mock_value}'
```

Comment:
my_output: "test and #{env}"

You could also rename 'test' to `input_key` to make it clearer